ext/session/session.c

```diff
@@ -1724,6 +1730,12 @@ static PHP_FUNCTION(session_set_cookie_params)
 		zend_string_release(ini_name);
 	}
 
+    if (argc > 5) {
```

Comment:
`if (samesite) {`

Testing the var is more readable and less likely to get screwed up because of some silly off-by-one error later on.